bin/jest.js

```diff
@@ -9,293 +9,4 @@
-        'Both --onlyChanged and a path pattern were specified, but these two ' +
-        'options do not make sense together. Which is it? Do you want to run ' +
-        'tests for changed files? Or for a specific set of files?'
-      );
-    }
-
-    if (argv.watchExtensions && argv.watch === undefined) {
-      throw new Error(
-        '--watchExtensions can only be specified together with --watch.'
-      );
-    }
-
-    if (argv.testEnvData) {
-      argv.testEnvData = JSON.parse(argv.testEnvData);
-    }
-  })
-  .argv;
-
-function runJest() {
-  if (argv.help) {
-    optimist.showHelp();
-
-    process.on('exit', function() {
-      process.exit(1);
-    });
-
-    return;
-  }
-
-  const cwd = process.cwd();
-
-  // Is the cwd somewhere within an npm package?
-  let cwdPackageRoot = cwd;
-  while (!fs.existsSync(path.join(cwdPackageRoot, 'package.json'))) {
-    if (cwdPackageRoot === '/' || cwdPackageRoot.match(/^[A-Z]:\\/)) {
-      cwdPackageRoot = cwd;
-      break;
-    }
-    cwdPackageRoot = path.resolve(cwdPackageRoot, '..');
-  }
-
-  // Is there a package.json at our cwdPackageRoot that indicates that there
-  // should be a version of Jest installed?
-  const cwdPkgJsonPath = path.join(cwdPackageRoot, 'package.json');
-
-  // Is there a version of Jest installed at our cwdPackageRoot?
-  const cwdJestBinPath = path.join(cwdPackageRoot, 'node_modules/jest-cli');
-
-  // Get a jest instance
-  let jest;
-
-  if (fs.existsSync(cwdJestBinPath)) {
-    // If a version of Jest was found installed in the CWD package, use that.
-    jest = require(cwdJestBinPath);
-
-    if (!jest.runCLI) {
-      console.error(
-        'This project requires an older version of Jest than what you have ' +
-        'installed globally.\n' +
-        'Please upgrade this project past Jest version 0.1.5'
-      );
-
-      process.on('exit', function() {
-        process.exit(1);
-      });
-
-      return;
-    }
-  } else {
-    // Otherwise, load this version of Jest.
-    jest = require('../');
-
-    // If a package.json was found in the CWD package indicating a specific
-    // version of Jest to be used, bail out and ask the user to `npm install`
-    // first
-    if (fs.existsSync(cwdPkgJsonPath)) {
-      const cwdPkgJson = require(cwdPkgJsonPath);
-      const cwdPkgDeps = cwdPkgJson.dependencies;
-      const cwdPkgDevDeps = cwdPkgJson.devDependencies;
-
-      if (cwdPkgDeps && cwdPkgDeps['jest-cli']
-          || cwdPkgDevDeps && cwdPkgDevDeps['jest-cli']) {
-        console.error(
-          'Please run `npm install` to use the version of Jest intended for ' +
-          'this project.'
-        );
-
-        process.on('exit', function() {
-          process.exit(1);
-        });
-
-        return;
-      }
-    }
-  }
-
-  jest.runCLI(argv, cwdPackageRoot, function(success) {
-    process.on('exit', function() {
-      process.exit(success ? 0 : 1);
-    });
-  });
-}
-
-runJest();
+require('./../src/cli').Run();
```

Comment:
Can you change your editor to add newlines at the end of a file? Seems like all of your files don't have it.